### ISSUE TITLE
Create user experience working group

### ISSUE THREAD

**Issue by xmatthewx** [MozillaFoundation/research#54]
Problem: loop between design, build, research, feedback, and iteration is tooooo long 

Solutions: TBD by User Experience working group 

Goals: 
 - Prep research with current UX questions
 - Schedule regular feedback debriefs
 - Synthesize and prioritize feedback for iteration
 - Coordinate
  - research workshops
  - remote testing
  - analytics
  - club feedback ??

Next steps:
 - schedule a meeting, of course
 - keep it simple

Potential names: 
 - ux shadow force
 - gta (gradient tucker alliance)
 - crystal clarity

in it to win it: @LauraReynal @flukeout @sabrinang @adamlofting @k88hudson

**Comment by cassiemc** [comment]
+me

**Comment by LauraReynal** [comment]
+ @thisandagain FYI

**Comment by LauraReynal** [comment]
Scheduled a Kick off meeting for the tuesday post-PTO
Adding @vazquez as well !

**Comment by adamlofting** [comment]
Just had a great chat with Dominik Strohmeier who runs metrics for Firefox OS, and was previously a user researcher on that team.

Their team are also working right now on how to join up Metrics, User Research, UX and design to build 'learning loops' (I like that title).

We're going to coordinate ways to share between MoFo and FxOS (both approach, and results).

**Comment by cassiemc** [comment]
Nice! Look forward to that @adamlofting. Sorry to miss the kickoff next week but will join subsequent meetings.

**Comment by LauraReynal** [comment]
Hey Folks, update from today's first UX JAM FORCE :jack_o_lantern: :strawberry: :strawberry: 
Notes from our meetings https://docs.google.com/document/d/1bYjJH_RD0TinKadOiI-mRsqIXE2-X1xal0_cNOQ6JA0/edit

TL;DR 
- Meet at the end of each heartbeat, on Wednesdays to discuss UX issues and User testing 
- Make sure we just to file issues, but also talk over the different observations and have each stakeholders reprensented during the jam force : metrics, research, design, engineers, roadmappers
- Do more frequent testing and rapid prototyping before we build a feature as opposed to only long trips


For our next meeting:
- @flukeout @LauraReynal and everyone interested will triage all issues with the feedback label
- @xmatthewx will prepare a deck for us to work on and have more visual playfulness 
- @sabrinang will sync with Hannah and see what she thinks and how this could help her
- @k88hudson will chat with engineers and see if they have specific needs regarding testing 

Jam on everyone, thanks for this first meeting !

**Comment by xmatthewx** [comment]
Thanks for the recap @LauraReynal! Here's a deck for gathering ideas and testing triage: http://bit.ly/ux-jam-force

**Comment by xmatthewx** [comment]
Design issue on visual triage and roadmapping: https://github.com/MozillaFoundation/Design/issues/55
